### PR TITLE
Updated ELK Stack

### PR DESCRIPTION
**Pull request by paurosello**
Updated components
Disabled gatekeeper for now as (POC)
Enabled curator

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# v0.2.0
+Upgraded all elastic components to 7.4.0
```
Comment (rossf7) [inline CHANGELOG.md]: Are there any upstream changelogs we should link to?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+Added curator with 3 days history
+The following namespaces are ommited for logging:
+- default
+- kube-system
+- kube-node-lease
+- kube-public
+- giantswarm
```
Comment (rossf7) [inline CHANGELOG.md]: Nit but what about ordering alphabetically?

File: helm/kubernetes-elastic-stack-elastic-logging/requirements.lock [hunk 3]
```diff
@@ -1,18 +1,18 @@
 dependencies:
 - name: elasticsearch
   repository: https://giantswarm.github.com/sample-catalog
-  version: 0.1.7
+  version: 0.2.1
```
Comment (rossf7) [inline helm/kubernetes-elastic-stack-elastic-logging/requirements.lock]: Are these in the sample catalog? If so I think we should move them.

@pipo02mix WDYT?

File: helm/kubernetes-elastic-stack-elastic-logging/requirements.yaml [hunk 4]
```diff
@@ -1,25 +1,30 @@
...
   version: "0.1.0"
   repository: https://giantswarm.github.com/sample-catalog
-  condition: elasticsearch-exporter.enabled
\ No newline at end of file
+  condition: elasticsearch-exporter.enabled
+
+- name: elasticsearch-curator
+  version: "2.0.3"
+  repository: https://giantswarm.github.com/sample-catalog
+  condition: elasticsearch-curator.enabled
```
Comment (rossf7) [inline helm/kubernetes-elastic-stack-elastic-logging/requirements.yaml]: Super nitpicky and existing bug but please add a newline here.

**Review by rossf7 — COMMENTED** [review]
Direction is good and my main query is whether we should move subcharts out of the sample catalog.

Also admin point. It's best to ask for reviews from teams and SIGs instead of individuals. Please request reviews from @giantswarm/team-batman on apps PRs.

**Comment by rossf7** [comment]
> Create an Ingress to access kibana limited to the IPs of happa...

That is a nice idea and each app CR has a cluster configmap that includes the base domain for the tenant cluster.

https://github.com/giantswarm/cluster-operator/blob/e36b36b14952f973cc1b656889288b44b34125e2/pkg/v21/resource/clusterconfigmap/desired.go#L30-L33

File: helm/kubernetes-elastic-stack-elastic-logging/requirements.lock (hunk 3 above)
Comment (pipo02mix) [reply helm/kubernetes-elastic-stack-elastic-logging/requirements.lock]: yeah think so too

Comment (paurosello) [reply helm/kubernetes-elastic-stack-elastic-logging/requirements.lock]: Where do I have to move them?

File: CHANGELOG.md (hunk 1 above)
Comment (paurosello) [reply CHANGELOG.md]: Added link to Upstream release notes

File: CHANGELOG.md (hunk 2 above)
Comment (paurosello) [reply CHANGELOG.md]: Done :)

File: helm/kubernetes-elastic-stack-elastic-logging/requirements.yaml (hunk 4 above)
Comment (paurosello) [reply helm/kubernetes-elastic-stack-elastic-logging/requirements.yaml]: Done

File: helm/kubernetes-elastic-stack-elastic-logging/requirements.lock (hunk 3 above)
Comment (rossf7) [reply helm/kubernetes-elastic-stack-elastic-logging/requirements.lock]: They should also be in the incubator catalog.

Comment (paurosello) [reply helm/kubernetes-elastic-stack-elastic-logging/requirements.lock]: There is an error with the architect that is not allowing me to publish the "subcharts" to the incubator catalog https://github.com/giantswarm/giantswarm/issues/7271 could we move forward with this so customer can try out the chart?

**Review by pipo02mix — APPROVED** [review]
LGTM if @rossf7 agree

File: helm/kubernetes-elastic-stack-elastic-logging/requirements.lock (hunk 3 above)
Comment (rossf7) [reply helm/kubernetes-elastic-stack-elastic-logging/requirements.lock]: Yes no need to delay this.

Let's move the subcharts to the incubator as a later step. We should also rename the app to `elastic-stack-app` to match the new naming convention.

**Review by rossf7 — APPROVED** [review]
LGTM

**Pull request merged**